robottests/test_remotescript.robot

```diff
@@ -2,31 +2,249 @@
 
 *** Settings ***
 
+Library    Process
+Library    OperatingSystem
+Library    Collections
 Library    crl.remotescript.RemoteScript    WITH NAME    RemoteScript
 
-Test Setup    Set RemoteScript Sudo Target
+Test Setup    Better Set RemoteScript Targets
```

Comment:
Maybe some more descriptive keyword name than **Better Set RemoteScript Targets**

Reply:
Changed the name to Better Set Sudo RemoteScript Targets to avoid double naming issue.